ipapython/ipaldap.py

```diff
@@ -379,8 +379,10 @@ def _set_raw(self, name, value):
                 name, value.__class__.__name__, value))
         for (i, item) in enumerate(value):
             if not isinstance(item, bytes):
-                raise TypeError("%s[%d] value must be str, got %s object %r" % (
-                    name, i, item.__class__.__name__, item))
+                raise TypeError(
+                    "%s[%d] value must be bytes, got %s object %r" % (
+                        name, i, item.__class__.__name__, item)
```

Comment:
nitpick, leave it consistent with other code in that module, can be refactored at once later